packages/yew/src/dom_bundle/braw.rs

```diff
@@ -0,0 +1,193 @@
+use wasm_bindgen::JsCast;
+use web_sys::Element;
+
+use crate::dom_bundle::bnode::BNode;
+use crate::dom_bundle::traits::{Reconcilable, ReconcileTarget};
+use crate::dom_bundle::utils::insert_node;
+use crate::dom_bundle::BSubtree;
+use crate::html::AnyScope;
+use crate::virtual_dom::VRaw;
+use crate::{AttrValue, NodeRef};
+
+pub struct BRaw {
+    pub html: AttrValue,
```

Comment:
I don't think there's a reason to keep the original html inside the attached bundle, just remove this field.